core/backend/src/IModelDb.ts

```diff
@@ -769,6 +770,7 @@ export abstract class IModelDb extends IModel {
    * [[include:IModelDb.updateProjectExtents]]
    * ```
    */
+

```

Comment:
Why are you (or your editor?) inserting blank lines between functions and their documentation?

Reply:
I think this may be the result of some SWB comments no being fully deleted. I will try to remove these accidental new lines.